.gitignore

```diff
@@ -4,6 +4,7 @@
 *.iws
 *.lock_message
 *.pyc
+__pycache__/
```

Comment:
I saw this show up in the v2 UI. I was surprised because `*.pyc` is supposed to cover it, but I figure there's little harm in adding this.

Reply:
Gitignore directives are purely based on filenames, right? So a `*.pyc` gitignore directive wouldn't match a filename `__pycache__` right?

Reply:
I think. `__pycache__/` is a folder, though.